.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Integration Tests
```

Comment:
```suggestion
name: Integration Test
```
Singular for now?

Reply:
its gonna be multiple tho eventually, right?

Reply:
We run one integration test for 2 cases, does that count as singular or plural? 🤔